testCommon/src/testFixtures/java/com/virtuslab/gitmachete/testcommon/TestFileUtils.java

```diff
@@ -20,7 +20,9 @@ private TestFileUtils() {}
   public static void copyScriptFromResources(String scriptName, Path targetDir) {
     URL resourceUrl = TestFileUtils.class.getResource("/" + scriptName);
     assert resourceUrl != null : "Can't get resource";
-    Files.copy(Paths.get(resourceUrl.toURI()), targetDir.resolve(scriptName), StandardCopyOption.REPLACE_EXISTING);
+    String scriptContents = IOUtils.resourceToString("/" + scriptName, StandardCharsets.UTF_8);
+    Files.copy(IOUtils.toInputStream(scriptContents, StandardCharsets.UTF_8), targetDir.resolve(scriptName),
```

Comment:
Oh! even better, without explicitly loading the file contents to the memory... +1